samples/sheets/insertColumn

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const {google} = require('googleapis');
+const sampleClient = require('../sampleclient');
+
```

Comment:
looks like you missed the extension on the filename, let's call this:

`insert-column.js`

If you don't mind?